kagiso_smart_404/tests/test_utils.py

```diff
@@ -2,7 +2,109 @@
 
 from wagtail.wagtailcore.models import Page
 
-from ..utils import suggest_page_from_misspelled_slug
+from ..utils import determine_if_slug_matches_one_page_exactly, suggest_page_from_misspelled_slug
+
+
+class DetermineIfSlugMatchesOnePageExactlyTest(TestCase):
+
+    def test_matching_slug_returns_single_page(self):
+        home_page = Page.objects.get(slug='home')
+        article = Page(
+            title='Workzone with Bridget Masinga',
+            slug='workzone-with-bridget-masinga'
+        )
+        home_page.add_child(instance=article)
+
+        result = determine_if_slug_matches_one_page_exactly(
+            '/shows/workzone-with-bridget-masinga/', home_page)
```

Comment:
/shows/workzone-with-bridget-masinga/ is the correct slug part (workzone with bridget), type the slug something like
"workzon_with_bridget" and it will fail as no trigrams are used